content/MLOps/105.md

```diff
@@ -61,19 +61,9 @@ If you have done the [Data fabric demo](https://cp4d-outcomes.techzone.ibm.com/d
 
 14. Select **Numeric/categorical** from the **Data type** dropdown. Select **Binary classification** from the **Algorithm type** dropdown. Click on the blue **Save and continue** button at the bottom.
 15. From the **Model details** screen, click on the **Edit** (pencil) icon on the **Training data** tile.
-16. From the **Location** dropdown, select **Cloud Object Storage**. For the **Resource instance ID** field that appears, copy and paste the following value:
+16. From the **Location** dropdown, select **Cloud Object Storage**. For the **Resource instance ID** field that appears, copy and paste the supplied values for resource ID and API key. 
```

Comment:
Technically we are good to keep the resource ID in the workshop content and share API key outside of the lab instructions.